lib/ansible/modules/network/avi/avi_actiongroupconfig.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+#
+# Created on Aug 25, 2016
+# @author: Gaurav Rastogi (grastogi@avinetworks.com)
+#          Eric Anderson (eanderson@avinetworks.com)
+# module_check: not supported
+# Avi Version: 16.3.2
+#
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import os
+# Comment: import * is to make the modules work in ansible 2.0 environments
+# from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import *
+
+HAS_AVI = True
+
+try:
+    from avi.sdk.utils.ansible_utils import (ansible_return, purge_optional_fields,
+        avi_obj_cmp, cleanup_absent_fields, avi_ansible_api)
+except ImportError:
+    HAS_AVI = False
+
+EXAMPLES = """
+- code: 'avi_actiongroupconfig controller=10.10.25.42 username=admin '
+            ' password=something'
+            ' state=present name=sample_actiongroupconfig'
+description: "Adds/Deletes ActionGroupConfig configuration from Avi Controller."
+"""
+
+DOCUMENTATION = '''
+---
+module: avi_actiongroupconfig
+author: Gaurav Rastogi (grastogi@avinetworks.com)
+
+short_description: ActionGroupConfig Configuration
+description:
+    - This module is used to configure ActionGroupConfig object
+    - more examples at <https://github.com/avinetworks/avi-ansible-samples>
+requirements: [ avisdk ]
+version_added: 2.3
+options:
+    controller:
+        description:
+            - location of the controller. Environment variable AVI_CONTROLLER is default
```

Comment:
All descriptions should be full sentences, e.g. Capital letters and full stops.

Variables such as AVI_CONTROLLER should be written as `C(AVI_CONTROLLER)` so they are formatted correctly.